### PR TITLE
[usecase-cordova][XWALK-3882] Add usecase for cordova xwalk host

### PR DESCRIPTION
**Pull request by zhuyongyong**
Impacted tests(desgined): new 1, update 0, delete 0
Unit test platform: [Android]
Unit test result summary: pass 0, fail 1, block 0

BUG=https://crosswalk-project.org/jira/browse/XWALK-3882

### CODE REVIEW

**Comment by Honry** [comment]
You can make it as designed before confirmed failure reason with developer.

File: usecase/usecase-cordova-android-tests/samples/CordovaXwalkHost/res/index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ARE DISCLAIMED. IN NO EVENT SHALL INTEL CORPORATION BE LIABLE FOR ANY DIRECT,
+INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+Authors:
+        Xu,Yuhan <yuhanx.xu@intel.com>
```
Comment (Honry) [inline usecase/usecase-cordova-android-tests/samples/CordovaXwalkHost/res/index.html]: Please update the copyright to up-to-date, and also the Author.

File: usecase/usecase-cordova-android-tests/tests.full.xml [hunk 2]
```diff
@@ -27,6 +27,8 @@
       </testcase>
       <testcase component="Crosswalk Use Cases/Cordova" execution_type="manual" platform="android" priority="P0" status="approved" type="functional_positive" id="AppContainer" purpose="AppContainer Test">
       </testcase>
+      <testcase component="Crosswalk Use Cases/Cordova" execution_type="manual" platform="android" priority="P0" status="approved" type="functional_positive" id="CordovaXwalkHost" purpose="CordovaXwalkHost Test">
```
Comment (Honry) [inline usecase/usecase-cordova-android-tests/tests.full.xml]: This test is still approved here.

File: usecase/usecase-cordova-android-tests/samples/CordovaXwalkHost/README.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+## Usecase Design
+
+This sample demonstrates cordova project app can set xwalk host, include:
+
+* '<access origin="*" />' in config of cordova project is available for setting xwalk host
```
Comment (Honry) [inline usecase/usecase-cordova-android-tests/samples/CordovaXwalkHost/README.md]: You mean it is default configuration? Have you try if it is failed without this tag?

Comment (zhuyongyong) [inline usecase/usecase-cordova-android-tests/samples/CordovaXwalkHost/README.md]: <access origin="*" /> is default configuation in cordova usecase. I just check it without the tag, but it is also pass.

File: usecase/usecase-cordova-android-tests/samples/CordovaXwalkHost/index.html [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+<script src="../../js/tests.js"></script>
+
+<body>
+  <div id="header">
+    <h3 id="main_page_title"></h3>
+  </div>
+  <div class="content">
+    <h4>Test Steps</h4>
+    <ol>
+      <li>According to http://otcqa.sh.intel.com/wiki/Cordova_Build_Steps to create a Cordova Example App suite</li>
```
Comment (Honry) [inline usecase/usecase-cordova-android-tests/samples/CordovaXwalkHost/index.html]: Please DON'T add internal link in test case.

File: usecase/usecase-cordova-android-tests/samples/CordovaXwalkHost/index.html [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+      <li>Add '<access origin="http://google.com" />' in the config.xml of the app suite</li>
+      <li>Add the following code in the index.html of the App Suite</li>
+        <textarea disabled="disabled" rows="25" cols="60">
```
Comment (Honry) [inline usecase/usecase-cordova-android-tests/samples/CordovaXwalkHost/index.html]: You create a index.html directly.

File: usecase/usecase-cordova-android-tests/tests.xml [hunk 6]
```diff
@@ -27,6 +27,8 @@
       </testcase>
       <testcase component="Crosswalk Use Cases/Cordova" execution_type="manual" id="AppContainer" purpose="AppContainer Test">
       </testcase>
+      <testcase component="Crosswalk Use Cases/Cordova" execution_type="manual" id="CordovaXwalkHost" purpose="CordovaXwalkHost Test">
```
Comment (Honry) [inline usecase/usecase-cordova-android-tests/tests.xml]: Isn't it designed test?

File: usecase/usecase-cordova-android-tests/samples/CordovaXwalkHost/index.html [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+<body>
+  <div id="header">
+    <h3 id="main_page_title"></h3>
+  </div>
+  <div class="content">
+    <h4>Test Steps</h4>
+    <ol>
+      <li>Create a Cordova Example App suite</li>
+      <li>Add '<access origin="http://google.com/" />' in the config.xml of the app suite</li>
```
Comment (Honry) [inline usecase/usecase-cordova-android-tests/samples/CordovaXwalkHost/index.html]: You miss steps to remove existing access meta-data.

File: usecase/usecase-cordova-android-tests/samples/CordovaXwalkHost/index.html [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+      <li>Copy usecase-cordova-android-tests/res/index.html to the www/index.html of the App Suite</li>
+      <li>Build apk file using the app suite</li>
+      <li>Install and Launch the app for the first time</li>
+      <li>Update access element to '<access origin="http://crosswalk-project.org/" />' in the config.xml of the app suite</li>
```
Comment (Honry) [inline usecase/usecase-cordova-android-tests/samples/CordovaXwalkHost/index.html]: Duplicate step?

**Comment by Honry** [comment]
LGTM

**Pull request merged**